autospec/files.py

```diff
@@ -241,6 +242,13 @@ def push_file(self, filename):
             (r"^/usr/share/info/", "info"),
             (r"^/usr/share/abi/", "abi"),
             (r"^/usr/share/omf", "main", "/usr/share/omf/*"),
+            (r"^/usr/lib64/openmpi/bin/", "openmpi"),
+            (r"^/usr/lib64/openmpi/share", "openmpi"),
+            (r"^/usr/lib64/openmpi/include/", "dev"),
+            (r"^/usr/lib64/openmpi/lib/[a-zA-Z0-9._+-]*\.so$", so_dest_ompi),
+            (r"^/usr/lib64/openmpi/lib/[a-zA-Z0-9._+-]*\.a$", "staticdev"),
+            (r"^/usr/lib64/openmpi/lib/[a-zA-Z0-9._+-]*\.so\.", "openmpi"),
+            (r"^/usr/lib64/openmpi/lib/", "dev"),
```

Comment:
So dev packages will not install the openmpi components, this means that if a user wants to develop against an openmpi library they'll need the corresponding openmpi bundle and the devpkg bundle where as normally the dev package would pull in the lib, bin components. This is a little bit broken since we expect a bundle to have all its dependencies to not have broken symlinks and the like.

Reply:
There is only a single  -dev package, containing also openmpi files, same file types as the non-openmpi (i.e. header files and *libs.so) . So the -dev package would include something like this:
<pre>
...
/usr/include/p4est.h
/usr/lib64/libsc-2.2.so
/usr/lib64/libsc.so
/usr/lib64/openmpi/include/p4est.h
/usr/lib64/openmpi/lib/libsc-2.2.so
/usr/lib64/openmpi/lib/libsc.so
...
</pre>
(For some reason github does not allow me to attach a file)

Reply:
I will make sure the -dev package will pull in -openmpi package (openmpi bin and libs are all in a single package)

Reply:
Great, thanks!